torch/distributions/multivariate_normal.py

```diff
@@ -182,9 +182,7 @@ def covariance_matrix(self):
 
     @lazy_property
     def precision_matrix(self):
-        # TODO: use `torch.potri` on `scale_tril` once a backwards pass is implemented.
-        scale_tril_inv = torch.inverse(self._unbroadcasted_scale_tril)
-        return torch.matmul(scale_tril_inv.transpose(-1, -2), scale_tril_inv).expand(
+        return torch.cholesky_inverse(self._unbroadcasted_scale_tril).expand(
```

Comment:
We still don’t have batching for cholesky_inverse (it’s in progress)

Reply:
Ah, my mistake! I should use cholesky_solve here...